docs/how-to.md

```diff
@@ -0,0 +1,4 @@
+# How to use..
```

Comment:
I think this branch might need a rebase to include the updated documentation if it's already in master, or have the other PR merged in then rebased here.

I've approved the earlier PR to merge in - would you please merge it in, then try rebasing this branch to include the updates docs from it.

https://github.com/thegreenwebfoundation/admin-portal/pull/231#pullrequestreview-942040899

If it isn't this would probably be better as a separate PR in future.